U.K. list cleanup

ksl1989 opened the pull request:
* 92 News UK
  * Changed language to Urdu
  * Added category
* Ahlulbayt TV
  * Added category
* Al Araby
  * Changed category -- general content according to Wikipedia
* Al-Hiwar
  * Removed duplicate link (with added GET parameters, makes no difference)
* BBC Arabic
  * Appended "(Backup)" to two of the streams' names, left the official stream as the main one.
* BBC Cbeebies
  * Added tvg-id & tvg-name
  * Added logo
  * Changed the name to "Cbeebies" (it's a BBC channel but not called BBC)
* BBC World News
  * Added tvg-id & tvg-name
  * Appended "(Backup)" to one of the streams' names, left as the main one the one the seems faster and has better quality.
* Cruise 1st TV
  * Added category
* English Club TV
  * Changed logo for larger one
  * Added category
* Flow Sports 1
  * Removed all links to this channel, according to Wikipedia and the official website this is Caribbean channel based in Jamaica
* Gemporia
  * Corrected the name to Gems TV
  * Removed one link which wasn't working.
  * (The remaining link also isn't working well for me.)
* Jatt TV bhangra music
  * Removed both links, neither is working
* KICC TV
  * Removed, link not working
* Mafa Shortfilm
  * Added category
  * Removed duplicate link (link to a stream already included in the multi-quality playlist above)
* MTA 1
  * Added category
  * Removed one link already included in the multi-quality playlist master.m3u8.
  * Replaced the other channels with the multi-quality playlist master.m3u8
  * Removed "English" from tvg-language because it seems it only broadcasts in English occasionally (But left Urdu in the Urdu one because that may be the language being broadcast)
  * Marked the different version in the channel name: Original, English, Urdu
  * Added "(Backup)" to a link in a different format, being unclear which version it is
* MTA 2
  * Removed one link already included in the multi-quality playlist master.m3u8.
  * Added category
  * Removed "English" from tvg-language because neither link was broadcasting in English
  * Replaced the other channel with the multi-quality playlist master.m3u8 & removed unnecessary GET parameters
* MTA 3
  * Removed three links already included in the multi-quality playlist master.m3u8.
  * Added category
* MTA Africa
  * Removed one link already included in the multi-quality playlist master.m3u8.
  * Added category
* Publitools
  * Removed this link because the channel is not from the U.K. Wikipedia says it's from Syria, LyngSat says it's from Jordan.
* Reuters
  * Removed this link because it's only broadcasting a static image with text. The Reuters TV service has been shut down, according to Wikipedia.
* Sheffield TV
  * Changed the name to Sheffield Live TV, which seems to be the official name
  * Added category
* Sky News
  * Removed two links which don't seem to work, the m3u8 file exists but the stream doesn't load.
* Sky News Arabia
  * Removed one link which is already included in the multi-quality playlist above.
* TG4
  * Removed link because this is an Irish channel that broadcasts mainly in Irish and is already in the Irish list
* Venus TV
  * Appended "(Backup)" to one of the streams. Leaving the official stream from their website as the main one.

ksl1989 committed:
UK list cleanup

* 92 News UK
  * Changed language to Urdu
  * Added category
* Ahlulbayt TV
  * Added category
* Al Araby
  * Changed category -- general content according to Wikipedia
* Al-Hiwar
  * Removed duplicate link (with added GET parameters, makes no difference)
* BBC Arabic
  * Appended "(Backup)" to two of the streams' names, left the official stream as the main one.
* BBC Cbeebies
  * Added tvg-id & tvg-name
  * Added logo
  * Changed the name to "Cbeebies" (it's a BBC channel but not called BBC)
* BBC World News
  * Added tvg-id & tvg-name
  * Appended "(Backup)" to one of the streams' names, left as the main one the one the seems faster and has better quality.
* Cruise 1st TV
  * Added category
* English Club TV
  * Changed logo for larger one
  * Added category
* Flow Sports 1
  * Removed all links to this channel, according to Wikipedia and the official website this is Caribbean channel based in Jamaica
* Gemporia
  * Corrected the name to Gems TV
  * Removed one like which wasn't working.
  * (The remaining link also isn't working well for me.)
* Jatt TV bhangra music
  * Removed both links, neither is working
* KICC TV
  * Removed, link not working
* Mafa Shortfilm
  * Added category
  * Removed duplicate link (link to a stream already included in the multi-quality playlist above)

ksl1989 committed:
MTA channels cleanup

* MTA 1
  * Added category
  * Removed one link already included in the multi-quality playlist master.m3u8.
  * Replaced the other channels with the multi-quality playlist master.m3u8
  * Removed "English" from because it tvg-language because it seems it only broadcasts in English occasionally (But left Urdu in the Urdu one because that may be the language being broadcast)
  * Marked the different version in the channel name: Original, English, Urdu
 * Added (Backup) to a link in a different format, being unclear which version it is

* MTA 2
  * Removed one link already included in the multi-quality playlist master.m3u8.
  * Added category
  * Removed "English" from because it tvg-language because neither link was broadcasting in English
  * Replaced the other channel with the multi-quality playlist master.m3u8 & removed unnecessary GET parameters

* MTA 3
  * Removed three links already included in the multi-quality playlist master.m3u8.
  * Added category

* MTA Africa
  * Removed one link already included in the multi-quality playlist master.m3u8.
  * Added category

ksl1989 committed:
UK list cleanup

* Publitools
  * Removed this link because the channel is not from the U.K. Wikipedia says it's from Syria, LyngSat says it's from Jordan.

* Reuters
  * Removed this link because it's only broadcasting a static image with text. The Reuters TV service has been shut down, according to Wikipedia.

* Sheffield TV
  * Changed the name to Sheffield Live TV, which seems to be the official name
  * Added category

* Sky News
  * Removed two links which don't seem to work, the m3u8 file exists but the stream doesn't load.

* Sky News Arabia
  * Removed one link which is already included in the multi-quality playlist above.

* TG4
  * Removed link because this is an Irish channel that broadcasts mainly in Irish and is already in the Irish list.

* Venus TV
  * Appended "(Backup)" to one of the streams. Leaving the official stream of the website as the main one.

freearhey commented on channels/uk.m3u:
```diff
@@ -1,115 +1,71 @@
 https://a.jsrdn.com/broadcast/22705/+0000/hi/c.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/ssybCGi.jpg" group-title="",Islam Channel
 http://59299831b66d0.streamlock.net/islam-live/islam.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hindi" tvg-logo="http://jatttv.c1.biz/wp-content/uploads/2020/03/cropped-jattv.png" group-title="",Jatt TV bhangra music
-https://edge.hls.hitbox.tv/hls/desijattuk/index.m3u8?st=JCp6gtouk2QPBi3gjVQaTQ;ci=lc5Dj7aG8uTMtaU_WwLCxg
```
Link is working

ksl1989 replied:
Now I can open it too, but this doesn't look like a real channel, more like some one trying something at home, and looking at the webpage, trying to grab some ad money. I also don't see anything so indicate it is from the U.K. But if you want it here I can add it back.

freearhey commented:
Could you elaborate on Flow Sports and Publitools. You just deleted the links to them or are going to move them to the corresponding playlists?

nepalicana commented:
BBC world news, one of the link is not working anymore which used to be in HD streaming :(
Newly added channels such as bbc 1,2,4 say limit exceeded while streaming.

freearhey commented:
@nepalicana Thank you for noticing! 

I've already deleted all links that have a limit (#1215)

ksl1989 committed:
Added Publitools

Originally in the U.K list. Country unclear, Wikipedia says it's from Syria, LyngSat says it's from Jordan.

ksl1989 commented:
@nepalicana 
> BBC world news, one of the link is not working anymore which used to be in HD streaming :(

Both links are working now, maybe there was a temporary issue.

ksl1989 commented:
@freearhey 
> Could you elaborate on Flow Sports and Publitools. You just deleted the links to them or are going to move them to the corresponding playlists?

I intended to add both to the corresponding list but I noticed now that Flow Sports 1 is a fake channel, it's not live streaming, it just shows the same video segment. It shows exactly the same whenever you open it.

I added Publitools to the unsorted list because, as I said, I'm not sure which country it belongs to.

ksl1989 committed:
Merge branch 'master' into master

nepalicana commented:
> @nepalicana 
> 
> > BBC world news, one of the link is not working anymore which used to be in HD streaming :(
> 
> 
> 
> Both links are working now, maybe there was a temporary issue.

Well I just happen to check the links the HD one is still not working:(

ksl1989 commented:
I resolved the conflict by adding all channels, thinking those were channels you added after my PR, now I'm thinking maybe those were channels you deleted. If that's right, maybe it's better to just accept this PR and then delete the channels again, rather than make this PR messier.

nepalicana commented:
> @nepalicana Thank you for noticing! 
> 
> 
> 
> I've already deleted all links that have a limit (#1215)

I really love your collections of channel but I’m sorry to say there is still the same channels which say limit exceeded

ksl1989 commented:
Let's have @freearhey  confirm and decide. Now I was also having trouble accessing that stream, it seems it's a bit unstable, before it was great. I think we should leave it for now and remove it later if it really goes offline.
